js/renovation/ui/grids/data_grid_light/types.ts

```diff
@@ -4,15 +4,32 @@
 import { JSXTemplate } from '@devextreme-generator/declarations';
 
 export type RowData = Record<string, unknown>;
+export type ColumnUserConfig = string;
+export type KeyExpr = string;
+export type Key = unknown;
+export interface Row {
+  key?: Key;
+
+  data: RowData;
+
+  rowType: string;
```

Comment:
```suggestion
  rowType: 'data' | 'detail';
```

Or maybe better `type Row = DataRow | DetailRow` and define `DataRow`, `DetailRow` separately